src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,67 @@
+import { useState, useEffect } from 'react';
+import React from 'react';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { PersonComponent } from '../PersonComponent';
+import { PersonType } from '../../types';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<PersonType[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState<string | null>(null);
+
+  useEffect(() => {
+    getPeople()
+      .then(data => {
+        setPeople(data);
+      })
+      .catch(() => {
+        setError('Something went wrong');
+      })
+      .finally(() => {
+        setIsLoading(false);
+      });
+  }, []);
+
+  if (isLoading) {
+    return <Loader />;
+  }
+
+  if (error) {
+    return (
+      <p data-cy="peopleLoadingError" className="has-text-danger">
+        {error}
+      </p>
+    );
+  }
+
+  if (!people.length) {
+    return <p data-cy="noPeopleMessage">There are no people on the server</p>;
```

Comment:
Too long. It will be more readable

```suggestion
    return (
      <p data-cy="noPeopleMessage">
        There are no people on the server
      </p>
    );
```

Reply:
i can't. Because linter.